docs/book/deployment.md

```diff
@@ -465,17 +467,23 @@ Combining the above examples will net you `cloudy cats dev` in en-GB and
 Once again, you can view the result skill invocation names in the deployed model files at
 `.deploy/{yourDeploymentTarget}/model-{locale}.json`.
 
+### Deployment Target-based Skill Configuration
+
+You can change the skill behavior and language model based on the deployment target you
+are executing tests or deploying the skill for. Please see [DEPLOY variables](
+/book/expressions.html#deploy-variables) for more information.
+
 ### Monetization
 
 Please see the [Monetization chapter](/book/monetization.html).
 
 ## Extra Note: Including `production` in the Deployment Target Name
 
 There is a special condition on deployment target names. If the name does *not*
-have the word `production` in it, a ` (development)` will be automatically
-appended to the name of your skill, with `development` specified as your named
-deployment target. We recommend labelling only your production skill deployment
-target(s) with `production`.
+have the word `production` in it, a ` (<target>)` will be automatically
+appended to the name of your skill, with `<target>` being the name of the
+deployment target you used to deploy the skill. We recommend labelling only
```

Comment:
trivial: "labelling" is Br. whereas "labeling" is Am. - think we've pretty consistently used Am. spelling throughout the docs.